content/en/boilerplates/snips/index.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
A really nice optimization would be to not generate a snip file for boilerplate.md files that don't have any `{{< text }}` snippets in them and then only source the .sh file if it exists.

We can leave that for a future PR, if you don't have time to do it now.

Reply:
Done already :) Number of files in the PR reduced from 31 to 23.